index.src.html

```diff
@@ -4256,6 +4303,16 @@
     to specify an offset and stride for each plane in the destination
     {{BufferSource}}. If unspecified, the planes will be tightly packed. It is
     invalid to specify planes that overlap.
+: <dfn dict-member for=VideoFrameCopyToOptions>format</dfn>
+:: A {{VideoPixelFormat}} for the pixel data in the destination
+    {{BufferSource}}. Explicity values cab be:
+    {{RGBA}}, {{RGBX}}, {{BGRA}}, {{BGRX}} and the same as {{VideoFrame/format}}.
```

Comment:
What happens if `frame.format` is `null`?

Reply:
"Explicity values cab" -> "Potential values are"
"and the same as" -> "and other values of"